opencog/cython/opencog/utilities.pyx

```diff
@@ -23,3 +25,19 @@ def finalize_opencog():
         c_finalize_python()
 
     is_initialized = False
+
+
+@contextmanager
+def tmp_atomspace():
+    """
+    Context manager, to create child atomspace from current default 
+    """
+    parent_atomspace = get_type_ctor_atomspace()
+    if parent_atomspace is None:
+        raise RuntimeError("Default atomspace is None")
+    atomspace = create_child_atomspace(parent_atomspace)
+    initialize_opencog(atomspace)
```

Comment:
We also need to make `initialize_opencog` go away. If you look at what it does, (`utilities.pyx`) it does two things: intialize the C++ python code, and call ` set_type_ctor_atomspace`. The first part -- initializing the C++ python code, that should happen automatically, when one does `from opencog.atomspace import foo` -- just make the `__init__()` method initialize the C++. 

I don't know if should ask you to do it, or just open another issue for this, or what; this already came up earlier in conversation with stellarspot Sorry to mention it again

Reply:
Ok, it's on my todo list.